<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>642</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="2">
     <widget class="QWidget" name="thumbnailBackGround" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(93, 93, 139);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QWidget" name="thumbnailLayout_container" native="true">
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>560</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1580</width>
           <height>1580</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(72, 116, 79, 100);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>42</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(56, 56, 84);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnMinimise">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 85, 127);
border:1px solid rgba(0,0,0,100);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMaximiseToggle">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 85, 127);
border:1px solid rgba(0,0,0,100);</string>
         </property>
         <property name="text">
          <string>[]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCloseProgram">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(202, 82, 82);
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border:1px solid rgba(0,0,0,100);</string>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="sideBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 85, 127);
border: 1.5px solid rgb(76, 76, 113);</string>
      </property>
      <widget class="QSlider" name="scaleSlider">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>21</width>
         <height>160</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="scaleSlider_label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>200</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>x4-cols</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
